delivery_recipe/quality_control_in_sprints.md

```diff
@@ -0,0 +1,78 @@
+# Quality Control in Sprints
+
+If a job’s worth doing, it’s worth doing properly. If we’re going to create quality products then there’s a lot more to it than simply testing at the end of the sprint cycle. Here’s what goes into driving quality throughout our work.
+
+## Coaching
+
+Right from the start of a change programme we need to be making the right decisions, asking the right questions and prioritising correctly. Ultimately this is the job of the Product Owner but because it’s such a big job we provide coaching and support to help them focus on what’s important.
+
+For many Product Owners, their time with us is also their first foray into Agile so there is a lot more going on for them to take in. We help them through that process, we provide the prompts, get them to prepare for meetings in advance, to have a clear picture of what they want to achieve and what’s most important.
+
+The [Coach](/delivery_recipe/the_coach.md) adds value by keeping the Product Owner focused on what’s important.
+
```

Comment:
Suggest we have something here on 'Definition of Done' for the work, and 'Acceptance Criteria' for each story... so that expectations are clear